package.json

```diff
@@ -49,12 +50,15 @@
     "imagemin-pngquant": "^6.0.1",
     "is-valid-glob": "^1.0.0",
     "journalize": "^2.5.1",
+    "prettier": "^2.8.8",
     "request": "^2.88.2",
     "sass": "^1.49.8"
   },
   "dependencies": {
     "@popperjs/core": "^2.11.2",
+    "ansi-colors": "^4.1.3",
     "bootstrap": "^5.1.3",
+    "fs-extra": "^11.1.1",
```

Comment:
These two packages were imported in some of the gulp file, but were not actually imported via `npm`.